Source/MonoGame.Extended/Particles/Profiles/Profile.cs

```diff
@@ -1,60 +1,48 @@
 ﻿using Microsoft.Xna.Framework;
+using MonoGame.Extended.Shapes;
 
 namespace MonoGame.Extended.Particles.Profiles
 {
     public abstract class Profile //: ICloneable
     {
-        public abstract void GetOffsetAndHeading(out Vector2 offset, out Axis heading);
+        public abstract void GetOffsetAndHeading(out Vector2 offset, out Vector2 heading);
 
-        public object Clone()
-        {
+        public object Clone() {
             return MemberwiseClone();
         }
 
         public enum CircleRadiation { None, In, Out }
 
-        public static Profile Point()
-        {
-            return new PointProfile();
+        public static Profile Point(PointF point) {
+            return new OutlineProfile(point);
         }
 
-        public static Profile Line(Axis axis, float length)
-        {
-            return new LineProfile { Axis = axis, Length = length };
+        public static Profile Line(LineF line) {
+            return new OutlineProfile(line);
         }
 
-        public static Profile Ring(float radius, CircleRadiation radiate)
-        {
-            return new RingProfile { Radius = radius, Radiate = radiate };
+        public static Profile Ring(CircleF circle, CircleRadiation radiate) {
+            return new OutlineProfile(circle, radiate, circle.Center);
```

Comment:
> This conflicts with the fact that the emitter also has a position.

While this is true, this also makes it so that you can have shapes that aren't exactly centered on the emitter, this can be useful for when working with IPathF shapes. It might also allow for an emitter to have multiple profiles spawning the same type of particles from differen positions.

> what does it mean that you've created a point profile from a PointF? 

Nothing more than it not requiring an explicit type for that.